packages/lesswrong/components/votes/AgreementVoteAxis.tsx

```diff
@@ -67,11 +67,16 @@ const AgreementVoteAxis = ({ document, hideKarma=false, voteProps, classes }: {
       </LWTooltip>
       
       <span className={classes.agreementScore}>
-        <LWTooltip title={<div>This {documentTypeName} has {karma} <b>agreement</b> karma ({voteCount} {voteCount === 1 ? "Vote" : "Votes"})</div>} placement="bottom">
```

Comment:
What was the exact bug here, that it showed agreement karma even when it was disabled?

Reply:
It showed agreement karma even when the hideCommentKarma flag was set, which was not the correct generalization of hideCommentKarma.